kustomize-sops.yaml

```diff
@@ -0,0 +1,97 @@
+package:
+  name: kustomize-sops
+  version: 4.3.3
+  epoch: 0
+  description: A Flexible Kustomize Plugin for SOPS Encrypted Resources
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - ca-certificates-bundle
+      - git
+      - kustomize
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/viaduct-ai/kustomize-sops
+      tag: v${{package.version}}
+      expected-commit: 294de9d6ac8c47a61d5ee14a79b6f216de65f872
+
+  - uses: go/build
+    with:
+      packages: ./ksops.go
+      output: ksops
+
+subpackages:
+  - name: ${{package.name}}-compat
+    description: Compat package for ${{package.name}}
+    pipeline:
+      - runs: |
+          mkdir -p "${{targets.contextdir}}"/usr/local/bin
+          ln -sf /usr/bin/ksops "${{targets.contextdir}}"/usr/local/bin/ksops
+          ln -sf /usr/bin/ksops "${{targets.contextdir}}"/usr/local/bin/kustomize-sops
```

Comment:
We should consider adding `stat` tests on symlinks here. (`stat /usr/local/bin/kustomize-sops`) and similarly for other one as well. 

ref: https://github.com/wolfi-dev/os/blob/b41e02edd1cd69d097de8ebde42be6ad7aa3d40c/linkerd2.yaml#L165